src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/InputLambdaBodyLengthSwitchExps.java

```diff
@@ -0,0 +1,34 @@
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.sizes.lambdabodylength;
+
+import java.util.stream.Stream;
+
+/* Config: default
+ */
+public class InputLambdaBodyLengthSwitchExps {
+
+    boolean method1(Nums k, String string) {
+        switch (k) {
+        case ONE -> { // ok
+            Stream.of(string.split(" "))
+                .anyMatch(Word -> "in".equals(Word));
+            System.out.println(1);
```

Comment:
please use line number of lamba in argument of method , it is really cool idea to make it more visual.

Reply:
done